pypeman/contrib/http.py

```diff
@@ -186,12 +186,14 @@ class HttpRequest(nodes.BaseNode):
         :param binary: bool, Get response content as bytes
         :param send_as_json: bool, If the method is a PATCH/POST/PUT, send data as json
         :param json: bool, Parse Json response content
+        :param add_meta: bool, If set, add resp infos in out message's meta (like headers)
```

Comment:
add_meta / add_to_meta / add_resp_to_meta ?

can close

Reply:
Mmm, at moment it add only headers to meta but i imagine we could want having more than just that